### PR TITLE
Drop travis and use gh actions instead

### PR DESCRIPTION
**Pull request by marc0olo**
includes changes of following PRs of @davidyuk: #350, #362 

also fixed the old compatibility tests that ran against the latest docker images

### CODE REVIEW

File: packages/aeproject-config/package-lock.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "name": "aeproject-config",
   "version": "2.2.0",
-  "lockfileVersion": 1
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
```
Comment (mradkov) [inline packages/aeproject-config/package-lock.json:7]: Is this supposed to be an empty string?

File: packages/aeproject-lib/package-lock.json [hunk 2]
```diff
@@ -1,8 +1,29 @@
 {
   "name": "aeproject-lib",
   "version": "2.2.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
```
Comment (mradkov) [inline packages/aeproject-lib/package-lock.json:7]: 🤔

File: packages/aeproject-logger/package-lock.json [hunk 3]
```diff
@@ -1,8 +1,68 @@
 {
   "name": "aeproject-logger",
   "version": "2.2.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
```
Comment (mradkov) [inline packages/aeproject-logger/package-lock.json:7]: 🤔

File: packages/aeproject-utils/package-lock.json [hunk 4]
```diff
@@ -1,8 +1,785 @@
 {
   "name": "aeproject-utils",
   "version": "2.2.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
```
Comment (mradkov) [inline packages/aeproject-utils/package-lock.json:7]: 🤔

File: packages/aeproject-config/package-lock.json (hunk 1 above)
Comment (marc0olo) [reply packages/aeproject-config/package-lock.json:7]: good question. this is the result I get when running `npm install` with npm version `7.12.1`

Comment (marc0olo) [reply packages/aeproject-config/package-lock.json:7]: @mradkov yep, it's about the new lockfile version -> https://nitayneeman.com/posts/catching-up-with-package-lockfile-changes-in-npm-v7/

**Review by mradkov — APPROVED** [review]

**Pull request merged**